native/swift/Tests/wordpress-api/LocalizationTests.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import Testing
+import WordPressAPI
+import WordPressAPIInternal
+
+class LocalizationTests {
+    @Test
+    func testParsingError() {
+        do {
+            let _ = try ParsedUrl.parse(input: "not-url")
+            Issue.record("Got an unexpected successful result")
+        } catch {
+            #expect(error.localizedDescription == "URL is invalid")
```

Comment:
Just in case it's unclear, this is this PR's end goal, where `error.localizedDescription` returns error messages for app users.